### PR TITLE
updated progressDeadline value to seconds format[600s]

### PR DESCRIPTION
**Pull request by vinayGudi**
Fixes #issue-number or description of the problem the PR solves

## Proposed Changes 

-
-
-

### CODE REVIEW

File: docs/serving/services/deployment.md [hunk 1]
```diff
@@ -42,5 +42,5 @@ metadata:
   namespace: knative-serving
 data:
 ...
-  progressDeadline: "10m"
+  progressDeadline: "600s"
```
Comment (csantanapr) [inline docs/serving/services/deployment.md:45]: Why are you changing to seconds? 

The description above mentions 10 minutes so "10m" seems more natural. 

is the description mentioned 600 seconds then using "600s" then is more natural. 

using such a hi number seconds in an example comes with cognitive load. 

if the example needs to in seconds then a value of 120s or less is easier to reason about.

Comment (vagababov) [reply docs/serving/services/deployment.md:45]: 👍 
I see no resaon for this change.

Comment (vinayGudi) [reply docs/serving/services/deployment.md:45]: but when i checked the actual deployed configmap yaml it is in minutes.
Screenshot from 2021-02-23 11-29-59

Comment (csantanapr) [reply docs/serving/services/deployment.md:45]: Ah I see perhaps would be good idea to update the configMap embedded docs to indicate that a and m are supported and change 600s to 10m

**Pull request closed without merging**